src/main/resources/com/axis/system/jenkins/plugins/downstream/yabv/BuildFlowAction/jobMain.groovy

```diff
@@ -1,3 +1,10 @@
 package com.axis.system.jenkins.plugins.downstream.yabv.BuildFlowAction
 
+link(rel: 'stylesheet',
+    type: 'text/css',
+    href: "${rootURL}/plugin/yet-another-build-visualizer/css/layout.css")
+
 include(my, 'buildFlow.groovy')
+
+script(src: "${rootURL}/plugin/yet-another-build-visualizer/scripts/ajax-build-flow.js",
```

Comment:
Don't you have to define buildFlowUpdateInterval here as well? Maybe move all JS related code into buildFlow?